doc/datavolumes.md

```diff
@@ -0,0 +1,136 @@
+# Data Volumes
+
+## Introduction
+Data Volumes(DV) are an abstraction on top of Persistent Volume Claims(PVC) and the Containerized Data Importer(CDI). The DV will monitor and orchestrate the upload/import of the data into the PVC. Once the process is completed, the DV will be in a consistent state that allow consumers to make certain assumptions about the DV in order to progress their own orchestration.
+
+Why is this an improvement over simply looking at the state annotation created and managed by CDI? For one, CDI has different state annotations for different operations (import vs cloning vs upload), where DVs have a single state field. This makes it a lot easier to check that one field vs trying to map all the different annotations in your process. Also DVs will always have a status field, where PVCs may or may not have a status annotation, so you will have to account for that as well.
```

Comment:
Actually, as of the controller dedupe, import/clone/upload all use the same annotation "cdi.kubevirt.io/storage.pod.phase".  Not sure this was actually a good idea.

The datavolume phases are a little more descriptive though. 

To me, the real (only?) value of DataVolume is the additional orchestration provided when used in conjunction with KubeVirt.

Reply:
Well the virt launcher controller uses the phase to determine if we can launch a VM, nothing stopping it from looking at the annotation. So now having a harder time spotting the benefits of the DV.

Reply:
The benefit of DataVolumes is it provides a versioned API projects like KubeVirt can integrate with.

Reply:
To clarify, I wouldn't try to justify DataVolumes outside of KubeVirt integration.